.github/workflows/test.yml

```diff
@@ -0,0 +1,361 @@
+name: test
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+      - '[0-9]+.[0-9]{2}'
+    tags:
+      - 'v*'
+  pull_request:
+
+env:
+  GO_VERSION: 1.18.5
```

Comment:
Oh; this needs to be 1.18.6 now, @crazy-max 

(or 1.19.1 if https://github.com/moby/moby/pull/43746 gets merged first)

Reply:
^^ 1.19.1 it is for master